webapp/views/launch/index.pug

```diff
@@ -77,6 +76,24 @@ html
       include ../table-attributes-modal/index.pug
 
     script.
+      //- enable Bootstrap tooltip on module select
+      $(function () {
+        const options = {
+          placement: 'bottom',
+          title: function () {
+            switch ($(this).val()) {
+              case 'module_1a': return 'The ‘Calculate time map’ module creates a map showing the time it takes to reach any part of the selected area from a given start point. Optionally, via points and/or affected areas with reduced speed can be defined.';
+              case 'module_2': return 'The ‘Query area’ module lets you filter map features based on a given set of constraints. For example, if a housing map layer is queried based on the household population and monthly income, only the houses within the given range will be returned.';
+            }
+          },
+          trigger: 'manual'
+        };
+        const target = $('#launch-module-menu');
+        target.tooltip(options);
+        target.on('focusin change', () => target.tooltip('show'));
+        target.on('focusout', () => target.tooltip('hide'));
+      });
```

Comment:
Just curious: does this work like IIFE?

Reply:
No, it's a shortcut for https://api.jquery.com/ready/